docs/docs/feature-library/activity-skill-filter.md

```diff
@@ -2,62 +2,144 @@
+### objective
+This feature aims to address these needs by providing the administrator the ability to lock down Activities by skill as well as ordering them based on a set order then alphabetically.
 
-## Configuration
+## Known Issues
 
-This plugin relies on custom configuration being applied to your underlying [Flex configuration](https://www.twilio.com/docs/flex/developer/ui/configuration#modifying-configuration-for-flextwiliocom). This is accomplished using the [Flex Configuration Updater](https://github.com/twilio-professional-services/twilio-proserv-flex-project-template/tree/main/flex-config) package in this repository.
+### issue one
+On older Twilio Flex accounts a TaskRouter Activity of "Reserved" may be present. This Activity is automatically hidden by the Flex UI and breaks the logic of the plugin. In this scenario it is advised you remove any TaskQueue references to the "Reserved" activity (which will likely be the activity to use when task is assigned on the TaskQueue and as such isn't actually used by Flex) and then delete the "Reserved" Activity.
 
-In your `ui_attributes.{environment}.json` file, enable the "activity_skill_filter" feature and add a rule for each activity SID you want to configure:
+## Flex User Experience
 
-```
+N/A
+
+## Configuration Options and Dependencies
+
+### configuration options
+
+```json
 "activity_skill_filter": {
   "enabled": true, // controls whether this feature is enabled
-  "filter_teams_view": true, // controls whether this feature affects the Teams view
+  "filter_teams_view": true, // controls whether this feature affects the Teams view WorkProfile activity menu
   "rules": {
-    "<ACTIVITY_SID>": {
+    "<ACTIVITY_NAME>": {
       "required_skill": "skill_name", // name of skill required to allow this activity, or null to allow all workers
       "sort_order": order // zero-based integer
     }
   }
 }
 ```
 
-As an example:
+This feature depends on a configured set of rules for the different Agent Activities. After [enabling](/setup-guides/configure_the_template#the-data-model) the feature the following configuration is required
 
-```
+The following are the descriptions of the configuration objects and what they represent 
+
+**filter_teams_view**: this variable represents whether or not to override the supervisor teams view, also applying the filters and sort order to the worker profile screen when viewing workers.
+
+#### rules
+
+this object allows the administrator to customize which activities are associated with which skills and what order they should appear in
+
+- **<ACTIVITY_NAME>**:  this element name represents the name of the activity for which you want to set a rule
+
+#### activity_name
+
+- **required_skill**: this variable represents the skill name that is required to see the activity in the drop down menu.
```

Comment:
Should also specify the ability to set it as null for all workers here too.

Maybe we should also explicitly call out that if you want nobody to see the activity, use a non-existent skill name.